.circleci/config.yml

```diff
@@ -39,6 +39,8 @@ jobs:
        - checkout
        - run: ci/do_circle_ci.sh bazel.coverage
        - run: ci/coverage_publish.sh
+       - store_artifacts:
+           path: /build/envoy/generated/coverage_tar/coverage.tar.gz
```

Comment:
did we consider uploading the entire path instead of a tar? i'm not quite sure what the ui will look like but if it presents it correctly it would be a lot easier.

Reply:
![image](https://user-images.githubusercontent.com/4712430/31102039-2c629c5a-a785-11e7-9882-1b1bb7a9b203.png)

i think it will work well judging from the circleci docs https://circleci.com/docs/2.0/artifacts/

Reply:
I can try that and see. I didn't get the impression it lets you view it inline. But it might.

Reply:
OK will try